Makefile

```diff
@@ -1,107 +1,113 @@
 #/bin/bash
 
-BASEDIR = $(shell pwd)
-SOURCE = "${BASEDIR}/lib"
-IMAGE = "elastica"
+.PHONY: update clean build setup start stop destroy run checkstyle checkstyle-ci code-browser cpd messdetector messdetector-ci dependencies phpunit test tests doc lint syntax-check loc phploc gource 
```

Comment:
I actually planned to clean up the .PHONY part by putting it on top of every command. Like this no "old" commands should appear here. In general I tried to reduce the number of targets we have in the makefile, but there is still quite a list :-)